File: Client/VRIS/app/src/main/java/com/valtech/amsterdam/vris/model/ITimeSlot.java
package com.valtech.amsterdam.vris.model;


import com.valtech.amsterdam.recyclist.modifiers.IHasId;

import org.joda.time.LocalDateTime;

/**
 * Created by marvin.brouwer on 19-7-2017.
 */

public interface ITimeSlot extends IHasId {

    int getId();

    LocalDateTime getStart();

    LocalDateTime getEnd();

    int getDurationInMinutes();

    boolean getSelected();
    void setSelected(boolean isSelected);
}

File: Client/app/src/main/java/com/labs/valtech/vris/models/ITimeslot.kt
package com.labs.valtech.vris.models


import org.joda.time.LocalDateTime

/**
 * Created by marvin.brouwer on 19-7-2017.
 */

interface ITimeslot {

    val id: String

    val startDate: LocalDateTime

    val endDate: LocalDateTime?

    val durationInMinutes: Int
}
